components/list/list-item-mixin.js

```diff
@@ -446,6 +476,7 @@ export const ListItemMixin = superclass => class extends ListItemDragDropMixin(L
 				<div class="d2l-list-item-active-border"></div>
 			</div>
 			${this._renderBottomPlacementMarker(html`<d2l-list-item-placement-marker></d2l-list-item-placement-marker>`)}
+			${this._displayKeyboardTooltip && tooltipForId ? html`<d2l-tooltip align="start" announced for="${tooltipForId}" for-type="descriptor">${this._renderTooltipContent()}</d2l-tooltip>` : ''}
```

Comment:
We use the `announced` attribute here since this tooltip could actually be pointing to the selection checkbox: `d2l-selection-input`, which contains `d2l-input-checkbox`, which contains `input`. This ensure that the tooltip gets announced even though the input being focused in in a different DOM scope.